ci-operator/jobs/openshift/odo/openshift-odo-main-presubmits.yaml

```diff
@@ -52,6 +52,106 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )psi-kubernetes-integration-e2e,?($|\s.*)
+  - agent: kubernetes
+    branches:
```

Comment:
Why `always_run:` is not there 🤔  . Did you run `make jobs` ?

Reply:
yes, we don't want to run tests on docs PR right, so we need to remove `always_run` and added `run_if_changed`.

Reply:
ok, could you please remove from other places too. Just to be in sync.

Reply:
I didn't understand by the term "other place"

Reply:
I am talking about other jobs. For ex https://github.com/openshift/release/pull/18247/files#diff-2a14d19c92854dd1baf93157739f79151bf0d5770aee591a1f0c41bf819863c6R4

Reply:
which one? I guess all of them have `run_if_changed`